docs/en/operations/settings/settings.md

```diff
@@ -1230,4 +1230,14 @@ Type: URL
 
 Default value: Empty
 
+## background_pool_size {#background_pool_size}
+
+Sets the number of threads performing background operations in table engines (for example, concurrent merges in `MergeTree`-engine tables). Applies at ClickHouse server startup and can't be changed in a user session. Adjusting this setting, you manage CPU and disk load. The lower value of the setting, the lower CPU and disk resources are required.
+
+Possible values:
+
+-   Any positive integer.
+
+Default value: 16.
```

Comment:
Also there are another pools:

- background_buffer_flush_schedule_pool_size
- background_move_pool_size
- background_schedule_pool_size
- background_distributed_schedule_pool_size

But I guess you saw them and this PR is only about `background_pool_size`

Reply:
Yes, this pull only about background_pool_size. I'll check other pulls, thank you :-)